src/pages/trader/index.js

```diff
@@ -140,38 +141,47 @@ function Trader() {
             card='summary_large_image'
             key="seo-wrapper"
         />,
-        <div className="page-wrapper" key={'page-wrapper'}>
-            <div className="trader-information-grid">
-                <div className="trader-information-wrapper">
-                    <h1>
-                        {trader.name}
-                        <img
-                            alt={trader.name}
-                            className={'trader-information-icon'}
-                            loading="lazy"
-                            src={`${process.env.PUBLIC_URL}/images/traders/${trader.normalizedName}-portrait.png`}
-                            onClick={() => openImageViewer(0)}
-                        />
-                    </h1>
-                    <span className="wiki-link-wrapper">
-                        <a href={`https://escapefromtarkov.fandom.com/wiki/${trader.normalizedName}`} target="_blank" rel="noopener noreferrer">
-                            {t('Wiki')}
-                        </a>
-                    </span>
-                    <p className='trader-details'>
-                        {trader.description}
-                    </p>
+        <div class="display-wrapper">
+          <div className={'entity-page-wrapper'} key={'trader-page-display-wrapper'}> 
+            <div className="entity-information-wrapper">
+              <div className="entity-top-content">
+                <img
+                  alt={trader.name}
+                  className={'entity-information-icon'}
+                  loading="lazy"
+                  src={`${process.env.PUBLIC_URL}/images/traders/${trader.normalizedName}-portrait.png`}
+                  onClick={() => openImageViewer(0)}
+                />
+                <div className="title-bar">
+                  <span class="type">
+                    Trader
```

Comment:
Should be translated too, like this:
```suggestion
                    {t('Trader')}
```